docs/source/gbd2017_models/concept_models/vivarium_sanofi_zenon/concept_model.rst

```diff
@@ -98,9 +98,40 @@ Utilization estimates used in this model are for the average number of outpatien
 
 .. _GHDx: http://ghdx.healthdata.org/record/ihme-data/UHC-cost-and-services-2016
 
+Initialization of patients into treatment for BAU
++++++++++++++++++++++++++++++++++++++++++++++++++
+
+We have distributions for the probability of being on Rx given high LDL-C and the probability of control given Rx.
+Using GBD data on high LDL-C (LDL-C > 4.9 mmol/L), we will have the population with measured LDL-C above the relevant threshold.  This, however, ignores the portion of the population that would have high LDL-C if they were not currently on medication.  
+To correct for this, we do the following:
+
+.. todo::
+
+	Add all equation components as probabilities
+
+:math:`\text{pop with high LDL if Rx did not exist=} \frac{\text{GBD estimate of pop with high LDL}} {\text{(1-prob(Rx|high LDL)} \times\ {\text{prob(control|Rx))}}}`
+
```

Comment:
per James's comment, let's write this as "prob(high LDL-C | no Rx) = ...

Reply:
@davidbernotas  We need to define 'controlled LDL-c' (is it 0? is it a certain threshhold?) 

TO-DO: Add into a table of definitions